pkg/reduce/data_forward.go

```diff
@@ -24,17 +24,16 @@ import (
 	"strconv"
```

Comment:
why are we renaming this file? This does much more than just `data_forward`

Reply:
I renamed it for 2 reasons:
1. `reduce/reduce.go` seems duplicate.
2. This file contains the following objects.
`type DataForward struct`
`func NewDataForward`
`func (d *DataForward) Start()`
`func (d *DataForward) forwardAChunk(ctx context.Context)`
They seem all tied with DataForward class. I agree it does more than just forwarding the data but I also feel renaming it to data_forward is more aligned with what we defined in the file.